.github/workflows/review-pr.yml

```
@@ -0,0 +1,65 @@
+name: Review PR
+on:
+  pull_request:
+    paths:
+      - 'recipes/**'
+
+# Explicitly set workflow permissions to avoid additional permissions being added from 
+# the Github UI.
+permissions:
+  contents: read
+
+env:
+  PR_NUMBER: ${{ github.event.number }}
+
+jobs:
+  # Check if the PR is the valid PR we can review.
+  check:
+    runs-on: ubuntu-latest
+    outputs:
+      status: ${{ steps.export.outputs.status }}
+    steps:
+    - name: Check valid PR
+      id: valid-pr
+      continue-on-error: true
+      uses: JJ/github-pr-contains-action@releases/v12.1
+      with:
+        github-token: ${{ github.token }}
+        bodyContains: '### Direct link to the package repository'
+
+    - name: Export result
+      id: export
+      if: steps.valid-pr.outcome == 'success'
+      run: |
+        echo "status=success" >> "$GITHUB_OUTPUT"
+
+  # Start the review process.
+  review:
+    needs: [check]
+    if: ${{ needs.check.outputs.status == 'success' }}
+    runs-on: ubuntu-20.04
+    permissions:
+      pull-requests: write
+    
+    steps:
+    - uses: purcell/setup-emacs@master
+      with:
+        version: 29.2
```

rationale: [parallel multi-version build](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)
```suggestion
        version: ${{ matrix.version }}
```